### DISCUSSION TITLE
Any forums or communities outside of here for MLX?

### DISCUSSION THREAD

**Discussion by fblissjr** [ml-explore/mlx#726] [General]
Not the biggest fan of Discord (but it has traction, so it works) - but also a big fan of community building and collaborative tinkering. MLX was one of the key reasons that got me to switch from 30+ years of dos->windows->wsl2/linux, and get myself an m2 ultra studio. Tons to explore from the ground up. Any communities out there that have started to gain traction in mlx development/exploration/hobbyist tinkering?

**Comment by awni** [comment]
If anyone has thoughts on this I would be keen to hear them!

A community discord could be fun to try as a starting point / experiment? Would you be interested in making one and adding me @fblissjr ?

**Comment by fblissjr** [comment]
> If anyone has thoughts on this I would be keen to hear them!
> 
> A community discord could be fun to try as a starting point / experiment? Would you be interested in making one and adding me @fblissjr ?

Done (sent you an email invite). 

I'm unfortunately not well versed enough in Discord to make running and moderating it successful - but more than happy to help support folks who are, and build a community around it.

**Reply by priontific** [reply]
Any chance you could drop that Discord link here? I'd love to join, but also just thinking that if anyone searches the forums for exactly this then it'd be good if the link for them to join was in this thread :)

**Reply by fblissjr** [reply]
Sure. Let me just get confirmation that @awni was able to join. Right now it's just me in there.

**Reply by awni** [reply]
I am in: https://discord.gg/ZKz3xgEKhv
